src/components/ideas/ideas.js

```diff
@@ -0,0 +1,121 @@
+        goal: "El objetivo de esta idea es terminar de implementar todas las validaciones del lenguage en la nueva versión del mismo y poder mostrarlas (en más de un idioma) en el IDE.",
+        url: linter,
+        mentors: [ fer, nicoS, alf ]
+    },
+    {
+        name: "Wollok - Autocompletado",
+        img: wollokImg,
+        context: linterContext,
+        goal: "El objetivo aquí es desarrollar un módulo de sugerencias de autocompletado en el servidor LSP para poder ser consumido desde el IDE. Esto implica hacer búsqueda de mensajes, referencias, clases, objetos o palabras claves dependendiendo del contexto.",
+        url: linter,
+        mentors: [ fer, nicoS, alf ]
+    },
+    {
+        name: "Wollok - Test Runner",
+        img: wollokImg,
+        context: linterContext,
+        goal: "Wollok ya cuenta con una forma de definir y correr tests automatizados dentro del lenguaje. El objetivo es servirlo como servicio LSP para poder ejecutar y visualizar los resultados de los tests desde el IDE.",
+        url: linter,
+        mentors: [ fer, nicoS, nahue ]
+    },
+    {
+        name: "Wollok - Consola",
+        img: wollokImg,
+        context: linterContext,
+        goal: "En la versión actual Wollok cuenta con una consola REPL que permite hacer pruebas manuales sobre el código. El objetivo es contar con esta misma herramienta en la nueva versión del servidor LSP para poder usarse desde los nuevos IDEs.",
+        url: linter,
+        mentors: [ fer, nicoS, nahue ]
+    },
+    {
+        name: "Wollok - Quick fixes & Refactors",
+        img: wollokImg,
+        context: linterContext,
+        goal: "El objetivo de esta idea es brindar ayuda para resolver los problemas detectados en el código (quick fixes) sugiriendo alternativas ya conocidas para editar el código de manera automática. De forma análoga, queremos ofrecer la posibilidad de hacer cambios al código de manera orgánica (refactors), como por ejemplo: renombrar algún elemento y sus usos, mover métodos entre entidades.",
+        url: linter,
+        mentors: [ fer, nicoS, alf ]
+    },
+    {
+        name: "Wollok - Diagramas",
+        img: wollokImg,
+        context: linterContext,
+        goal: "Actualmente Wollok cuenta con varios diagramas que permiten visualizar el estado de los objetos vivos (dinámico) y del código que se está desarrollando (estático). El objetivo es tener estas dos herramientas, que ayudan en la compresión de los conceptos, disponibles para VSCode.",
+        url: linter,
+        mentors: [ fer, nicoS, alf ]
+    },
+    {
+        name: "Wollok - Debugger",
+        img: wollokImg,
+        context: linterContext,
+        goal: "El objetivo aquí es implementar un módulo en el servidor LSP que ofrezca la posibilidad de debuggear la ejecución de un programa. Actualmente la implementación del lenguaje ya cuenta con una API para poder debuggear su ejecución y tenemos hecha una prueba de concepto sobre esto.",
```

Comment:
```suggestion
        goal: "El objetivo aquí es implementar un módulo en el servidor LSP que ofrezca la posibilidad de debuggear la ejecución de un programa. Actualmente la implementación del lenguaje ya cuenta con una API para poder debuggear su ejecución y tenemos hecha una prueba de concepto.",
```

no se si aclarar que la prueba de concepto está funcionando, o tirar un link...